__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require('../app.js');
+const request = require('supertest');
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {topicData, userData, articleData, commentData} = require("../db/data/test-data");
```

Comment:
you can just require it in as one big object as it destructured in the seed.js file